src/store/sessionTimer.ts

```
@@ -0,0 +1,64 @@
+
+  function updateSessionDate(date: Date): void {
+    sessionStartDate.value = date
+  }
+
+  const sessionTime = computed(() => 
+    `${pad(hoursSession.value % 24)}:${pad(minutesSession.value % 60)}:${pad(secondsSession.value % 60)}`
+  )
+
+  function updateSessionTimer(): void {
+    const now = new Date();
+    if (!(sessionStartDate.value instanceof Date) || isNaN(sessionStartDate.value.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)